docs/apm/transactions.asciidoc

```diff
@@ -4,9 +4,10 @@
 TIP: A {apm-overview-ref-v}/transactions.html[transaction] describes an event captured by an Elastic APM agent instrumenting a service.
 The APM agents automatically collect performance metrics on HTTP requests, database queries, and much more.
 
-// Clicking *service* brings you to detail of transaction
-Selecting a <<services,*service*>> will display all associated *transactions*.
-The charts and table on this dashboard display the transaction duration, requests per minute, and a list of transactions for the selected service.
+Selecting a <<services,*service*>> brings you to the *transactions* overview.
+The *transaction duration* and *requests per minute* chart display information on all transactions associated with the selected service.
+The *Transactions* table, however, provides only a list of _transaction groups_ for the selected service.
+In other words, this view groups all transactions of the same name together, and only displays each like transaction once.
```

Comment:
I feel "like" here is a typo or not needed?

Reply:
It wasn't a typo, but you're definitely right, it's a confusing way to word it. I don't think we just remove "like" because then it says each transaction is displayed once. I've updated it to hopefully be less confusing.